src/Components/ArticlePage.js

```diff
@@ -0,0 +1,56 @@
+import "./ArticlePage.css";
+import { useParams, useLocation } from "react-router-dom";
+import { getArticleById } from "../Api/Api";
+import React, { useEffect, useState } from "react";
+
+const ArticlePage = (props) => {
+  const { id } = useParams();
+  const { state } = useLocation();
+  const [article, setArticle] = useState(
+    state?.article || props.article || null
+  );
+  const [articleComments, setArticleComments] = useState([]);
+  const date = article.created_at.substring(0, 10);
```

Comment:
This line is causing an error when you navigate to the path directly - when article is `null`